resources/definitions/lotmaxx_sc20.def.json

```diff
@@ -0,0 +1,64 @@
+{
+    "name": "Lotmaxx SC-20",
+    "version": 2,
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "author": "lotmaxx.com",
+        "manufacturer": "Lotmaxx",
+        "file_formats": "text/x-gcode",
+        "platform": "lotmaxx_sc_10_20_platform.stl",
+        "has_materials": true,
+        "machine_extruder_trains":
+        {
+            "0": "lotmaxx_sc20_extruder_0"
+        }
+    },
+    "overrides": {
+        "machine_name": { "default_value": "Lotmaxx SC-20" },
+        "machine_width": { "default_value": 235 },
+        "machine_depth": { "default_value": 235 },
+        "machine_height": { "default_value": 320 },
+        "machine_head_polygon": { "default_value": [[-32,11],[-32,-32],[28,-32],[28,11]] },
```

Comment:
Same as above, you can use `machine_head_with_fans_polygon`.

Reply:
Thanks for your reply.